src/codeu/chat/common/Hub.java

```diff
@@ -1,147 +0,0 @@
-// Copyright 2017 Google Inc.
```

Comment:
I would like to avoid having multiple ways to do the same thing as it will only add extra complexity to the code base. The Timeline can do everything the Hub can do (and more) so there is really no need to keep the Hub around.